en/mkdocs.yml

```diff
@@ -332,23 +332,21 @@ nav:
         - 'Rule Based Provisioning': using-wso2-identity-server/rule-based-provisioning.md
         - 'Provisioning Patterns': using-wso2-identity-server/provisioning-patterns.md
       - 'Account Management Policies':
-        - 'Account Management Policies': using-wso2-identity-server/account-Management-Policies.md
```

Comment:
why has this been removed?

Reply:
Had only the section contents. So removed the page since there's a navigation available.